javascript-light-bulb/main.js

```diff
@@ -0,0 +1,22 @@
+/*
+get access to the elements with the class button-moon
+get access to the elements with the class button-sun
+get access to the elements with the class container-moon
+get access to the elements with the class container-sun
+create a variable that counts the amount of clicks on the button
+*/
+
+var $button = document.querySelector('.button-moon');
+var $container = document.querySelector('.container-moon');
+
+$button.addEventListener('click', flickSwitch);
+
+function flickSwitch(event) {
+  if ($container.className !== 'button container-sun') {
```

Comment:
Now your logic is dependent on the state of the DOM to know what to do, which is less than ideal. You should follow my initial suggestion of using a variable containing a boolean value to keep track of if the light is currently on or not, and use that variable's value to decide what action to take when the lightbulb is clicked.